### PR TITLE
Feature/12 teacher info component

### LINKED ISSUE #12: TeacherInfoComponent
**DoD**
- Display the email of the user.
- Display the username of the user.
- Use existing vuex UserModule

Screenshot 2019-11-18 at 15 11 41

### PR DESCRIPTION
**Pull request by TE4-jacob-klaren**
# TeacherInfoComponent Pull Request

## What issue did you implement or fix?
Fixed #12 

## Summary
- Added template for 1 teacher card
- Fixed compatibility with future feature #14 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

## How can this implementation be improved?
- Change bar into zar. :shipit:

### CODE REVIEW

File: frontend/src/components/TeacherInfoComponent.vue [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<template>
+    <v-card class="mx-auto" max-width="344" raised>
```
Comment (te4-gabriel-stark) [inline frontend/src/components/TeacherInfoComponent.vue]: Borde det inte vara en v-list-item istället för ett v-card? Så ser det kanske lite finare ut? Man kan göra en TeacherInfoListComponent som loopar igenom alla lärare som är en v-list som kallar på TeacherInfoComponent.

https://vuetifyjs.com/en/components/lists#lists

File: frontend/src/components/TeacherInfoComponent.vue [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+@Component({
+    components: {
+    }
+})
+export default class TeacherInfoComponent extends Vue {
+    newPwd: string = ""
+    @Prop({default: "No value provided"}) email: string
+    @Prop({default: "No value provided"}) name: string
+
+    private passwordReset() {
```
Comment (TE4-Daniel-Kull) [inline frontend/src/components/TeacherInfoComponent.vue]:
```suggestion
    private passwordReset() { // TODO: Implement "updatePassword" functionality in UserModule
```

Comment (TE4-Daniel-Kull) [reply frontend/src/components/TeacherInfoComponent.vue]: :P

**Comment by TE4-Daniel-Kull** [comment]
no

**Pull request closed without merging**